lightning.install

```diff
@@ -9,6 +9,34 @@ use Drupal\Core\Entity\Entity\EntityViewDisplay;
 use Drupal\field\Entity\FieldConfig;
 use Drupal\lightning_core\ConfigHelper as Config;
 
+/**
+ * Alerts users of Libraries API dependencies.
+ *
+ * A future version of Lightning will remove Libraries API, so
+ * users must add that dependency to composer.json manually.
+ */
+function lightning_requirements() {
+    $requirements = [];
+    $libraries_dependencies = [];
+    $enabled_modules = \Drupal::moduleHandler()->getModuleList();
+    
+    foreach ($enabled_modules as $module => $data) {
+        $info = \Drupal::service('extension.list.module')->getExtensionInfo($module);
+        if (in_array('drupal:libraries', $info['dependencies'])) {
```

Comment:
Three things: we should always pass TRUE as a third argument to in_array(), the actual name of the project will be `libraries:libraries` rather than `drupal:libraries`, and we also need to support the old syntax (just `libraries`, with no project prefix).